CampApp/Modules/CampSelection/Presenter/CampSelectionPresenter.swift

```diff
@@ -66,7 +67,6 @@ final class CampSelectionPresenter: CampSelectionPresenterProtocol {
     viewController.bungalowLabel.textAlignment = .center
     
   }
-  
   public func setupViews(viewController: CampSelectionController) {
     
```

Comment:
buralar böyle olmaz controller presenter içine yollanmaz. buraları cıntrollerda private func olarak kalsın. sonra yapmamız için buralara "//Fixme:" yaz